docs/adr/0001-provide-mechanism-to-show-hide-sections-of-accessibility-statement.md

```diff
@@ -0,0 +1,111 @@
+
+## Pros and Cons of the Options <!-- optional -->
+
+### [option 1]
+
+* Good, because no further work required.
+* Bad, because it doesn't solve the problem.
+
+### [option 2]
+
+`AboutThisService.scala` fragment would need to be updated so that it can take a new property for displaying VOA specific content. Currently, the section of content is static and comes directly from the internationalised messages property file with key `using-service.this-service-or-app-is-run`
+
+`ContactInformation.scala` fragment would need to be updated to take a flag property and the content below
+```html
+<p class="govuk-body">@{messages("in-person-contact.text-relay")}</p>
+<p class="govuk-body">@{messages("in-person-contact.bsl")}</p>
+```
+would need to either be displayed or not dependent on that new flag property.
+
+* Good, because the change would not be breaking to existing service yaml files as we are introducing new properties and not changing the existing view model dramatically.
+* Bad, because it is a less flexible approach, and means other team's in the future who need to make changes, will introduce multiple different properties that would eventually become confusing and less maintainable.
+
+### [option 3]
+
+Example of a new accessibility statement view model, this would introduce view models specific for each fragment within the accessibility statement.
+```scala
+case class AccessibilityStatement(
+  // common properties specific to the accessibility statement can be on the root level independently 
+  mobilePlatform: Option[MobilePlatform],
+  aboutThisService: Option[AboutThisService] = None,
+  complaintsProcess: Option[ComplaintsProcess] = None,
+  contactInformation: Option[ContactInformation] = None,
+  howAccessibleIsThisService: Option[HowAccessibleIsThisService] = None,
+  howServiceWasTested: Option[HowServiceWasTested] = None,
+  howToGetExtraSupport: Option[HowToGetExtraSupport] = None,
+  introduction: Option[Introduction] = None,
+  reportingAccessibilityProblems: Option[ReportingAccessibilityProblems] = None,
+  technicalDetails: Option[TechnicalDetails] = None
+) 
+```
+
+Example of service yaml that will be inline with the new accessibility statement view model.
+```yaml
+# common properties specific to the accessibility statement can be on the root level independently 
+aboutThisService:
+  # specific properties for fragment
+complaintsProcess:
+  # specific properties for fragment
+contactInformation:
+  # specific properties for fragment
+howAccessibleIsThisService:
+  # specific properties for fragment
+howServiceWasTested:
+  # specific properties for fragment
+howToGetExtraSupport:
+  # specific properties for fragment
+introduction:
+  # specific properties for fragment
+reportingAccessibilityProblems:
+  # specific properties for fragment
+technicalDetails:
+  # specific properties for fragment
+```
```

Comment:
Thoughts on this:

- We should consider how, if we make these YAML files configureable, we would expect teams to know what values are valid. Should it be an enum-like structure of allowed values? 
- Should it be flexible enough to allow HTML to be passed in (probably not, but worth verifying). 
- Rather than specific sections, should be view model allow passing in information about the type of statement, and then the business logic of which sections to display lives within accessibility-statement-frontend